src/coreComponents/mesh/EdgeManager.cpp

```diff
@@ -64,371 +66,29 @@ EdgeManager::~EdgeManager()
 
 void EdgeManager::resize( localIndex const newSize )
 {
+  // FIXME My tests say that this line could be commented out.
+  //       But those tests are not exhaustive.
```

Comment:
Remove the comment if no longer useful, or write something more specific about what you mean

Reply:
done, it's still useful!